lib/ansible/executor/module_common.py

```diff
@@ -1319,7 +1319,25 @@ def modify_module(module_name, module_path, module_args, templar, task_vars=None
     return (b_module_data, module_style, shebang)
 
 
-def get_action_args_with_defaults(action, args, defaults, templar):
+def get_action_args_with_defaults(action, args, defaults, templar, redirected_names=None):
+    collection_groups = {
+        'amazon.aws': 'aws',
```

Comment:
```suggestion
        'community.crypto': 'acme',
```
With this, my tests actually work :)